src/V3AstNodeOther.h

```diff
@@ -2427,6 +2427,14 @@ class AstFunc final : public AstNodeFTask {
     ASTGEN_MEMBERS_AstFunc;
     bool hasDType() const override { return true; }
 };
+class AstProperty final : public AstNodeFTask {
+    // A property inside a module
+public:
+    AstProperty(FileLine* fl, const string& name, AstNode* stmtp)
+        : ASTGEN_SUPER_Property(fl, name, stmtp) {}
+    ASTGEN_MEMBERS_AstProperty;
+    bool hasDType() const override { return true; }
+};
```

Comment:
Why do we inherit from `AstNodeFTask` if we don't use any task features? Also, could we just use `AstPropClocked`? Weird to have two node types with almost the same name, especially since this one doesn't do much new.

Reply:
Because Property references are parsed as AstFuncRef and linking rules are the same. Matching of arguments is also the same.
I think that I should find a better name for AstPropClocked. I think that we should have separate classes to distinguish between whole property (now AstProperty) and its expression (now AstPropClocked).